firebase/database.rules.json

```diff
@@ -133,6 +133,9 @@
             ".write": "auth != null && auth.uid == $uid"
           }
         },
+        "userGroups": {
+          ".write": "auth != null"
+        },
```

Comment:
Is this needed? It looks like you define it again a few lines below.

Reply:
We need this, because one is for overall usergroup and one is for the membership